js/src/forum/components/Composer.js

```diff
@@ -157,7 +157,7 @@ export default class Composer extends Component {
    * Draw focus to the first focusable content element (the text editor).
    */
   focus() {
-    this.$('.Composer-content :input:enabled:visible:first').focus();
+    this.state.editor.focus();
```

Comment:
This changes the behavior from focusing on the first available field (e.g. title) to always focusing on the post content.